source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -544,14 +547,24 @@ class WordDocument(UIADocumentWithTableNavigation,WordDocumentNode,WordDocumentB
 	def event_textChange(self):

```

Comment:
I think this can simply call event_caret. This also ensures brailleInput and vision are updated correctly.

Reply:
Thanks. I am about to change it in next commit.